### ISSUE TITLE
Glide Version Update

### ISSUE THREAD

**Issue by themounthead** [jibon57/nativescript-mediafilepicker#94]
This is more of a request than a bug report.

I'm using the nativescript-image-cache-it plugin alongside the mediafilepicker. It took me a while to realize that the incompatibility between the Glide version between both plugins was the reason why the picker was not loading.

- By removing the image-cache-it plugin altogether, the filepicker works
- If I install a much older version of the image-cache-it plugin (which has Glide Ver 4.8), the filepicker works

Current status :
Image-Cache-It :: Glide Version 4.10.0
MediaFilePicker:: Glide Version 4.8.0

Simply put, I don't know who to bring both plugins to play nicely. 

Seeking your advice.

Thanks
Vijay

**Comment by jibon57** [comment]
@themounthead Please try latest version & give feedback.
